File: submissions/cpp/461.cpp
class Solution {
public:
    int hammingDistance(int x, int y) {
        int sum = 0;
        while(x != 0 || y != 0) {
            int i = x % 2, j = y  % 2;
            cout << i << \u0027 \u0027 << j << endl;
            sum += (i ^ j == 1 ? 1 : 0);
            cout << sum << endl;
            x /= 2;
            y /= 2;
        }
        return sum;
    }
};

File: submissions/python/461.py
class Solution(object):
    def hammingDistance(self, x, y):
        """
        :type x: int
        :type y: int
        :rtype: int
        """
        x = bin(x^y)
        l = len(x)
        sum = 0
        for i in range(l):
            if x[i] == \u00271\u0027:
                sum += 1
        return sum